Header      header

uint64      size_total    # total size of disk in bytes
uint64      used          # bytes used
